tests/storage/restore.go

```diff
@@ -1828,7 +1845,7 @@ var _ = SIGDescribe("VirtualMachineRestore Tests", func() {
 					// TODO: consider ensuring network clone gets done here using StorageProfile CloneStrategy
 					dataVolume := libdv.NewDataVolume(
 						libdv.WithPVCSource(sourceDV.Namespace, sourceDV.Name),
-						libdv.WithStorage(libdv.StorageWithStorageClass(snapshotStorageClass), libdv.StorageWithVolumeSize("1Gi")),
+						libdv.WithStorage(libdv.StorageWithStorageClass(forcedHostAssistedScName), libdv.StorageWithVolumeSize("1Gi")),
```

Comment:
how come here it used the snapshot sc and you changed it to the forcedHostAssistedScName? was it a mistake?

Reply:
no, it was doing cross storage class clone from rwofilesystemclass -> snapshotclass to achieve host assisted artifically but now all DVs will be using the same storage class that's forced through an API to do host assisted

Reply:
oh ok got you